app/styles/ui/rebase-status.scss

```diff
@@ -0,0 +1,68 @@
+.rebase-status-component {
+  .rebase-status-icon-container {
+    height: 20px;
+    position: relative;
+    text-align: center;
+
+    .rebase-status {
+      background: var(--background-color);
+      border-width: 0 var(--spacing-half);
+      border: solid transparent;
+      box-sizing: content-box;
+      position: relative;
+      z-index: 1;
+    }
+  }
+
+  .rebase-status-loading {
+    color: $yellow-700;
+  }
+
+  .rebase-status-conflicts {
+    color: var(--color-conflicted);
+  }
+
+  .rebase-status-invalid {
+    color: $red-500;
+  }
+
+  .rebase-status-clean {
+    color: $green-500;
```

Comment:
Could we extract his into a CSS variable such that it's themeable please? Something like `--status-success-color` would be nice.

Reply:
Resolved in e63092a3102296c8e0d4d41006c6e514288d06a8